lib/polygeist/Passes/LoopRestructure.cpp

```diff
@@ -82,8 +78,6 @@ func @kernel_gemm(%arg0: i32, %arg1: memref<?xf64>) {
 
 using namespace mlir;
 
```

Comment:
We should figure out why this err'd -- specifically we do still want the debug type to be "LoopRestructure" for this pass.